src/modules/NtpClient.rb

```diff
@@ -282,6 +296,38 @@ def GetNtpServersByCountry(country, terse_output)
       end
     end
 
+    def read_ntp_conf
+      if !File.exist?(NTP_FILE)
+        log.error("File #{NTP_FILE} does not exist")
+        return false
+      end
+
+      begin
+        @ntp_conf = CFA::NtpConf.new
+        @ntp_conf.load
+      rescue
+        log.error("Failed to read #{NTP_FILE}")
```

Comment:
if you log message, it should be useful. I think at least write error message here to log would be very helpful as it can be bad permissions, corrupted disk, malformed file or whatever else.